core/main/src/broker/rules_engine.rs

```diff
@@ -176,15 +176,16 @@ pub fn jq_compile(input: Value, filter: &str, reference: String) -> Result<Value
     // start out only from core filters,
     // which do not include filters in the standard library
     // such as `map`, `select` etc.
-    let mut defs = ParseCtx::new(Vec::new());
 
+    let mut defs = ParseCtx::new(vec!["fromjson".into()]);
```

Comment:
Do we need to keep adding other commands in this list ?  If yes, it would be better to move this on a configuration file.

Reply:
+1 for configuration we need lots of other definitions to be supported as well